README.md

```diff
@@ -68,3 +68,35 @@ export interface raExpressMongooseOptions<T> {
   ACLMiddleware?: (name: string) => RequestHandler;
 }
 ```
+## Query Operators
+
+MongoDB Query Operators can be used by appending them as a suffix to the field name ([see here](https://marmelab.com/react-admin/FilteringTutorial.html#filter-operators)).
+### Supported Operators
+| MongoDB Query Operator | Field Suffix | Description                                                                                                                                                                     |
+|------------------------|--------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| $eq                    | _eq          | Matches values that are equal to a specified value. <br/> (Usefull for matching exact values if a field is included in allowedRegexFields) <br/> Example:  `/user?name_eq=Alex` |
+| $gt                    | _gt          | Matches values that are greater than a specified value. <br/> Example:  `/user?createdAt_gt=2022-10-25T00%3A00%3A00.000Z`                                                       |
+| $gte                   | _gte         | Matches values that are greater than or equal to a specified value. <br/> Example:  `/user?createdAt_gte=2022-10-25T00%3A00%3A00.000Z`                                          |
+| $in                    | _in          | Matches any of the values specified in an array. <br/> Example: `/user?name_in=Alex&name_in=Peter`                                                                              |
+| $lt                    | _lt          | Matches values that are less than a specified value. <br/> Example:  `/user?createdAt_lt=2022-10-25T00%3A00%3A00.000Z`                                                          |
+| $lte                   | _lte         | Matches values that are less than or equal to a specified value. <br/> Example:  `/user?createdAt_lte=2022-10-25T00%3A00%3A00.000Z`                                             |
+| $ne                    | _ne          | Matches all values that are not equal to a specified value. <br/> Example:  `/user?name_ne=Alex`                                                                                |
+| $nin                   | _nin         | Matches none of the values specified in an array. <br/> Example:  `/user?name_nin=Alex&name_nin=Peter`                                                                          |
+
+### Example Filter in React Admin
```

Comment:
I think a dedicated examples file would be really helpful here to lessen the clutter in the main README.md. If you could, consider updating the demo to include some of the capabilities here, it'd be great.